packages/nextjs/app/builders/0x872f1fFdeEe35a6074658B4FB185E2582Db7896A/page.tsx

```diff
@@ -0,0 +1,102 @@
+import Image from "next/image";
+import { NextPage } from "next";
+import {
+  AcademicCapIcon,
+  CodeBracketIcon,
+  RocketLaunchIcon,
+  StarIcon,
+  TrophyIcon,
+  XMarkIcon,
+} from "@heroicons/react/24/outline";
+import { BuidlGuidlLogo } from "~~/components/assets/BuidlGuidlLogo";
+import { Address } from "~~/components/scaffold-eth";
+
+const CleanerPage: NextPage = () => {
+  return (
+    <div className="w-full max-w-6xl mx-auto rounded-3xl shadow-2xl flex flex-row my-10 animate-gradient-x">
+      {/* image column */}
+      <div className="w-1/2 rounded-l-3xl bg-secondary">
+        <div className="flex justify-center items-center h-full">
+          <div className="w-80 h-80 overflow-hidden rounded-full shadow-lg">
+            <Image
+              src="/thumbnail.jpg"
+              alt="Cleaner Avatar"
+              className="object-cover w-full h-full"
+              width={500}
+              height={500}
+            />
+          </div>
+        </div>
+      </div>
+
+      {/* content column */}
+      <div className="w-1/2 bg-gradient-to-r from-secondary to-base-100 dark:from-secondary dark:to-base-300 rounded-r-3xl flex flex-col justify-center p-8">
+        <div className="flex justify-between items-center">
+          <h1 className="text-5xl font-semibold">
+            <span className="bg-clip-text text-black dark:text-white bg-gradient-to-r from-base-400 to-base-100 via-base-150 dark:from-secondary dark:via-secondary dark:to-secondary">
+              Cleaner
+            </span>
+            <span className="space-y-1">
+              <Address address="0x872f1fFdeEe35a6074658B4FB185E2582Db7896A" />
+            </span>
+          </h1>
+          <div className="flex gap-3">
+            <a href="https://github.com/cleanerzkp" target="_blank" rel="noopener noreferrer">
+              <img
```

Comment:
The github [actions lint fails](https://github.com/BuidlGuidl/batch5.buidlguidl.com/actions/runs/9052761727/job/24884748222?pr=16) due to use html img tag 

Lets disable the eslint for this lines by adding `{/* eslint-disable-next-line @next/next/no-img-element */}` to avoid mentioning all the remotePatterns in `next.config.js`

I think having just `avatars.githubusercontent.com` in `next.config.js` is fine since it can utilized by others too.